vue-app/src/components/ContributionModal.vue

```diff
@@ -118,10 +121,31 @@ export default class ContributionModal extends Vue {
   voteTxError = ''
   error = ''
 
+  mounted() {
```

Comment:
I know this is somewhat outside the scope of this PR but should we also add a check to see if they've already approved (& move to step 2 if they have)? Or does that complicated things if their allowance is now different?

Reply:
Ah, never mind, I see we already handle that 😄  https://github.com/ethereum/clrfund/blob/develop/vue-app/src/components/ContributionModal.vue#L159

Reply:
Uhmmm that's a good point. You are talking about #L180 right? where we have this condition:
```js
if (allowance < total) {
```